tech.json

```diff
@@ -1172,7 +1272,7 @@
               "requires": [
                 "canCrystalFlash",
                 "canCeilingClip",
-                "Morph"
+                "canMidAirMorph"
```

Comment:
oh, this looks like it shouldnt be there either, for artificial morph + spring ball applications